runtime/altair/src/lib.rs

```diff
@@ -1666,8 +1666,32 @@ impl_runtime_apis! {
 			let pool = pallet_pool_system::Pool::<Runtime>::get(pool_id)?;
 			pool.tranches.tranche_currency(tranche_loc).map(Into::into)
 		}
+
+		fn portfolio_valuation(pool_id: PoolId) -> Option<Balance> {
+			if !pallet_pool_system::Pool::<Runtime>::get(pool_id).is_some() {
+				return None;
+			}
+
+			match pallet_loans::Pallet::<Runtime>::update_nav_of_pool(pool_id) {
+				Ok((_, value)) => Some(value),
+				Err(_) => None,
+			}
+		}
+
+		fn max_borrow_amount(pool_id: PoolId, loan_id: ItemId) -> Option<Balance> {
```

Comment:
I'm going to make use of my pedantic new role 😝:

`Result` has an `ok()` to return a `None`, when there is an `Err`. Maybe it can be applied here. The whole function could be something like: 
```rust
fn max_borrow_amount(pool_id: PoolId, loan_id: ItemId) -> Option<Balance> {
    pallet_pool_system::Pool::<Runtime>::get(pool_id)?;
    pallet_loans::Pallet::<Runtime>::get_max_borrow_amount(pool_id, loan_id).ok()
}
```